music_assistant/server/providers/spotify/__init__.py

```diff
@@ -135,10 +145,13 @@ async def handle_async_init(self) -> None:
         """Handle async initialization of the provider."""
         self._cache_dir = CACHE_DIR
         self._ap_workaround = False
-        # try to get a token, raise if that fails
         self._cache_dir = os.path.join(CACHE_DIR, self.instance_id)
         # try login which will raise if it fails
         await self.login()
+        if self.config.get_value(CONF_CLIENT_ID):
```

Comment:
Better change the throttling params before `login()`, it might speed it up a bit.